src/entry-points/content/AllMediaElementsController.ts

```diff
@@ -77,6 +78,17 @@ const controllerTypeDependsOnSettings = [
   'experimentalControllerType',
   'dontAttachToCrossOriginMedia',
 ] as const;
+
+function isPerTabDisabledSync(): boolean {
+  try {
+    const key = getPerTabKeySync('perTabEnabled');
+    const stored = localStorage.getItem(key);
```

Comment:
This uses the website's local storage. We should not modify the site's local storage.